build.gradle

```diff
@@ -569,12 +569,6 @@ task integrationTest(type: Test) {
     }
 }
 
-tasks.named("integrationTest") {
```

Comment:
Won't this change also affect the normal test runs in the PR CI jobs?

Reply:
This was added recently in an effort to stabilize the `integrationTest` suite to get all tests passing on first run w/o retry. We added the `integrationTest` task as a dependency on `integTest` ([PR](https://github.com/opensearch-project/security/pull/4814)) which triggered these tests to be run with the release tests and there are a few tests failing on first try as you pointed out [here](https://github.com/opensearch-project/security/issues/4800).

PRs related to stabilizing `integrationTest` task:

- https://github.com/opensearch-project/security/pull/4844
- https://github.com/opensearch-project/security/pull/4851
- https://github.com/opensearch-project/security/pull/4855

^ this was a new addition. IMO we should follow-up to track flaky tests in the `integrationTest` sourceSet and see if this block reduces the number of flaky tests.

Reply:
@nibix There is an issue running [IPBruteForceAttacksPreventionTests](https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/security/IpBruteForceAttacksPreventionTests.java) on my mac. In order to get the tests to run I need to run commands like:

```
sudo ifconfig lo0 alias 127.0.0.2 netmask 0xff000000
```

for all IPs declared here: https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/security/IpBruteForceAttacksPreventionTests.java#L51-L61

Reply:
Added this block back in to increase maxHeapSize <- this does help stabilize the tests.

Reply:
>  IMO we should follow-up to track flaky tests in the integrationTest sourceSet and see if this block reduces the number of flaky tests.

Absolutely!

> There is an issue running [IPBruteForceAttacksPreventionTests](https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/security/IpBruteForceAttacksPreventionTests.java) on my mac.

Oh, I did not know that. :-( Using different loopback addresses was a kind of a nice feature to test IP address specific configuration. Still, looking at RFC 3330, MacOS seems to be the outlier here.